README.md

```diff
@@ -104,10 +104,11 @@ $ rubycritic --help
 Analyzer Configuration
 -------------------------
 
-* `Reek`: `RubyCritic` just uses `Reek`'s default [configuration loading mechanism](https://github.com/troessner/reek#configuration-file).
-
-This section is work in progress at the moment and is missing the other analyzers that RubyCritic supports.
-You can expect this to be updated soon.
+* [`Reek`](https://github.com/troessner/reek): `RubyCritic` utilizes `Reek`'s default [configuration loading mechanism](https://github.com/troessner/reek#configuration-file).
+  This means that if you have an existing `Reek` configuration file, you can just put this into your
+  project root and `RubyCritic` will respect this configuration.
+* [`flay`](https://github.com/seattlerb/flay): We use `flay`'s default configuration.
+* [`flog`](https://github.com/seattlerb/flog): We use `flog`'s default configuration with a couple of [smaller tweaks](https://github.com/whitesmith/rubycritic/blob/master/lib/rubycritic/analysers/helpers/flog.rb#L5).
```

Comment:
And no, i have zero idea what those smaller tweaks mean since `flog` has not really documented those :laughing: 
Skimming through flogs codebase it looks "kind-of" default-ish though, so I'd rather be a little vague here than document flog's internal. @guilhermesimoes maybe you still remember why you added those tweaks i link to above?